### PR TITLE
Prep to publish v1.2.0

### PR DESCRIPTION
**Pull request by nshahan**
* Allow builder to use new `outputStyle` option.
* Remove unused dev dependencies.
* CHANGELOG and README updates.

### CODE REVIEW

**Comment by nshahan** [comment]
@alademann Can you test with this branch?  Will this work for #32?

EDIT: oops maybe @aaronlademann-wf

**Comment by alademann** [comment]
@nshahan it works!  Thanks!!!

**Review by luisvt — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -82,6 +90,25 @@ dev_dependencies:
...
+
+```yaml
+targets:
+  $default:
+    builders:
+      sass_builder:
+        options:
+          outputStyle: compressed
+```
+
```
Comment (luisvt) [inline README.md:110]: in what file should we put this options? `build.yaml` or `sass_builder.build.yaml`?

**Comment by alademann** [comment]
> EDIT: oops maybe @aaronlademann-wf

@nshahan same person.  One is a personal account, the other is my Workiva account :)

File: README.md (hunk 1 above)
Comment (nshahan) [reply README.md:110]: It's recommended to use `build.yaml` to configure the option for your package. `sass_builder.yaml` is more like an "escape hatch" to override the sass_builder settings for all packages in your build that depend on it, including transitive dependencies.

**Pull request merged**